packages/x-data-grid/src/models/configuration/gridConfiguration.ts

```diff
@@ -7,7 +7,9 @@ export interface GridAriaAttributesInternalHook {
 
 export interface GridInternalHook
   extends GridAriaAttributesInternalHook,
-    GridRowAriaAttributesInternalHook {}
+    GridRowAriaAttributesInternalHook {
+  useCSSVariables: () => { id: string; variables: Record<string, string | number> };
```

Comment:
This hook is supposed to be provided by the dev when using a non-MUI design system, correct?
If so, we need to type `variables` accordingly.